common/src/main/java/de/maxhenkel/voicechat/config/ClientConfig.java

```diff
@@ -52,109 +52,109 @@ public ClientConfig(ConfigBuilder builder) {
-                        "The maximum amount of audio packets that are held back if a packet arrives out of order or gets dropped",
-                        "This prevents discarding audio packets that are slightly out of order",
+                        "The maximum number of audio packets that should be held back if a packet arrives out of order or is dropped",
+                        "This prevents audio packets that are only slightly out of order from being discarded",
                         "Set this to 0 to disable"
                 );
         deactivationDelay = builder
                 .integerEntry("voice_deactivation_delay", 25, 0, 100,
                         "The time it takes for the microphone to deactivate when using voice activation",
-                        "A value of 1 means 20 milliseconds, 2=40 ms, 3=60 ms, ..."
+                        "A value of 1 means 20 milliseconds, 2=40 ms, 3=60 ms, and so on"
                 );
         microphone = builder
                 .stringEntry("microphone", "",
                         "The microphone used by the voice chat",
-                        "Empty for default device"
+                        "Leave blank to use the default device"
                 );
         speaker = builder
                 .stringEntry("speaker", "",
-                        "The speaker used by the voice chat",
-                        "Empty for default device"
+                        "The speakers used by the voice chat",
+                        "Leave blank to use the default device"
                 );
         muted = builder
                 .booleanEntry("muted", true,
-                        "If the microphone is muted (only when using voice activation)"
+                        "Specifies whether the microphone is muted (only relevant for the voice activation method)"
                 );
         disabled = builder
                 .booleanEntry("disabled", false,
-                        "If the voice chat is disabled (sound and microphone off)"
+                        "Specifies whether the voice chat is disabled (sound and microphone off)"
                 );
         hideIcons = builder
                 .booleanEntry("hide_icons", false,
-                        "If the voice chat icons should be hidden"
+                        "Specifies whether the voice chat HUD, group chat HUD, and other in-game icons should be hidden"
                 );
         showGroupHUD = builder
                 .booleanEntry("show_group_hud", true,
-                        "If the group HUD should be visible"
+                        "Specifies whether the group chat HUD should be visible"
                 );
         showOwnGroupIcon = builder
                 .booleanEntry("show_own_group_icon", true,
-                        "If the own icon should be shown when in a group"
+                        "Specifies whether your player icon should be displayed in the group chat HUD when you are in a group"
```

Comment:
Specifies whether your **own** player icon should be displayed in the group chat HUD when you are in a group

Sounds better I think

Reply:
Fair enough, I'll add this.